python/py-peakutils/Portfile

```diff
@@ -19,11 +19,11 @@ long_description    ${description}
 
 homepage            https://peakutils.readthedocs.io
 
-checksums           rmd160  fb665e5e469b9251648c39095c71de02760248b7 \
-                    sha256  390696b7ffc7933ff1426d40fcb3079a79b99fcdf1f64e4122fce82584d77b00 \
-                    size    118684
+checksums           rmd160  0502dde7cba8e2c8056a8e950ed21d399f759166 \
```

Comment:
thanks for removing the manual setting of `master_sites` and `distfiles`; however, you'll need to revert the `checksums` back to the original I had before as well to have this to work. You might have (re-calculated) the checksums locally and it probably didn't complain, but that's likely because it had the distfile already downloaded from the bitbucket source you initially put in. So if you change something in where to grab the sources form you need to make sure to start from a clean slate, that can be done with `sudo port clean --all py\*-peakutils` (whether the escape `\` should be there depends on your shell).